en-US/ranking/criteria.md

```diff
@@ -2,78 +2,173 @@
+* **The total file size of the mapset must not exceed 50 MB.**
 
-## Mandatory
+## Background
 
-**You must follow all the requirements listed in this section. Failing to meet all the requirements will not allow your map to receive ranked status.**
+#### Rules
 
-### Metadata
+* **Every map in the mapset must have a background image.** Using different background images for different maps is acceptable.
+* **The resolution of a background image must be at least 1280x720.**
+* **The file size of a background image must not exceed 4 MB.**
+* **The background must be of exceptionally high quality.** 
+* **the background must be appropriate for players of all ages and should not contain explicit content.**
 
-The following pertains to metadata that your maps contain such as song artists, titles, and tags.
+#### Guidelines
 
-* **The artist and title must be absolutely correct.**
-* **The artist and title must be romanized.** However, non-romanized characters are allowed to be used in source and tags.
-* **The artist, title and source must not repeat in tags.** However, romanizing or translating the source in tags is encouraged, in case it's relevant.
-* **Tags must contain the genre of the song.**
-* **Every difficulty must have a difficulty name.**
+* The background should show relevance to the song in some way. While mappers are allowed creative freedom when it comes to choosing backgrounds, completely unrelated background images that do not compliment the song in any way should generally not be used.
 
-### Media
+## Audio
 
-The following pertains to media files such as background images and audio files that are contained in your mapset.
+#### Rules
 
-* **The resolution of background images must be at least 1280x720** and of exceptionally high quality. The file size of a background image must not exceed 4 MB.
-* **The mapset must contain only one audio file.** Multiple song file mapsets are not eligible to be ranked.
+* **The mapset must contain only one audio file.**  Mapsets with multiple audio files are not eligible to be ranked.
 * **MP3 is the only allowed audio file format.**
-* **The maximum bitrate allowed for audio files is 192kbps.**
-* **The total file size must not exceed 50 MB.**
+* **The maximum bitrate allowed for MP3 files is 192kbps.** 
+* **The audio must be appropriate for players of all ages and should not include content that is offensive just for the sake of being offensive.**
 
-### Maps
+# Maps
+
+#### Rules
 
 The following requirements are for the individual maps themselves.
 
 * **Autoplay must be able to achieve a 100% (X-Grade) score.**
 * **The maps must not have overlapping notes.**
-* **All long note releases must have at least 36 milliseconds of space after them.** The value is derived from 2x of the Standard "Marvelous" timing window.
 * **The maps cannot have more than 30 seconds of consecutive break time.**
```

Comment:
Apparently github doesn't highlight the line I'm talking about so this is in reference to the break time thing not the long note release thing

Reply:
I also think moving the break time rule to be a guideline is a good idea. As a rule, it must be enforced with no room for flexibility; as a guideline it can still be enforced while allowing for RS discretion.

Reply:
this requires the automod to be adjusted